examples/light-switch-app/silabs/SiWx917/src/AppTask.cpp

```diff
@@ -247,17 +244,19 @@ void AppTask::SwitchActionEventHandler(AppEvent * aEvent)
     }
 }
 
-void AppTask::ButtonEventHandler(const sl_button_t * buttonHandle, uint8_t btnAction)
+void AppTask::ButtonEventHandler(uint8_t button, uint8_t btnAction)
 {
-    if (buttonHandle == NULL)
-    {
-        return;
-    }
-
     AppEvent button_event           = {};
     button_event.Type               = AppEvent::kEventType_Button;
     button_event.ButtonEvent.Action = btnAction;
-
-    button_event.Handler = SwitchActionEventHandler;
-    sAppTask.PostEvent(&button_event);
+    if (button == 1)
```

Comment:
Please use a constant or define with a significant name to represent the button ID/name instead or just 1 or 0

Reply:
sure